### PR TITLE
dialog.submit sends an empty array in case of no appIds

### PR DESCRIPTION
**Pull request by lakhveerk**
Sending an empty array is less error prone than sending `undefined` in an array.

### CODE REVIEW

File: packages/teams-js/src/public/dialog.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export namespace dialog {
     ensureInitialized(FrameContexts.content, FrameContexts.sidePanel, FrameContexts.task, FrameContexts.meetingStage);
 
     // Send tasks.completeTask instead of tasks.submitTask message for backward compatibility with Mobile clients
-    sendMessageToParent('tasks.completeTask', [result, Array.isArray(appIds) ? appIds : [appIds]]);
+    sendMessageToParent('tasks.completeTask', [result, appIds ? (Array.isArray(appIds) ? appIds : [appIds]) : []]);
```
Comment (lakhveerk) [inline packages/teams-js/src/public/dialog.ts:82]: I cannot think of any scenarios, that can break v1 compatibility with this change. If you do, please let me know! :)

Comment (jadahiya-MSFT) [reply packages/teams-js/src/public/dialog.ts:82]: This shouldn't break back compat, but I would check to see what is expected from the hosts perspective. Maybe they are expecting an `undefined` in an array? Maybe it's better to just send `undefined`?

**Review by jadahiya-MSFT — DISMISSED** [review]

File: packages/teams-js/src/public/dialog.ts (hunk 1 above)
Comment (MarcinJuraszek) [reply packages/teams-js/src/public/dialog.ts:82]: Should we add a UT around this?

Comment (lakhveerk) [reply packages/teams-js/src/public/dialog.ts:82]: Good idea! Added a unit test for this!

**Review by jadahiya-MSFT — APPROVED** [review]

**Review by MarcinJuraszek — APPROVED** [review]

**Pull request merged**